### DISCUSSION TITLE
Espresso Machine PID controller using ESP32 and RainMaker · Espressif Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#192] [General]
# Espresso Machine PID controller using ESP32 and RainMaker · Espressif Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/espresso-machine/

**Comment by Pijnacker** [comment]
Hi Raffael,
Is it possible to get a more detailed parts list?
